src/main/java/org/ow2/proactive/catalog/dto/BucketMetadata.java

```diff
@@ -25,30 +25,61 @@
  */
 package org.ow2.proactive.catalog.dto;
 
+import java.util.Objects;
+
 import org.ow2.proactive.catalog.repository.entity.BucketEntity;
+import org.springframework.hateoas.ResourceSupport;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
 
-import lombok.EqualsAndHashCode;
+import lombok.Data;
 
 
 /**
  * @author ActiveEon Team
  */
-@EqualsAndHashCode(callSuper = true)
-public class BucketMetadata extends NamedMetadata {
+@Data
```

Comment:
Can you double check if you need to call the superclass equals and hashCode because: (at)EqualsAndHashCode(callSuper = true) was replace with  (at)Data